profiler/profiler_test.go

```diff
@@ -440,6 +445,8 @@ func TestCorrectTags(t *testing.T) {
 }
 
 func TestTelemetryEnabled(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
```

Comment:
CI hosts can be surprisingly slow. I think a longer timeout would be better -- 5 seconds? Just to reduce the odds of this test flaking.